ee/ucp/kubernetes/storage/use-azure-file.md

```diff
@@ -0,0 +1,247 @@
+---
+title: Configuring Azure File Storage for Kubernetes
+description: Learn how to add persistent storage to your Docker Enterprise clusters running on Azure with Azure File.
+keywords: Universal Control Plane, UCP, Docker EE, Kubernetes, storage, volume
+redirect_from:
+---
+
+Operators can provide persistent storage for workloads running on Docker
+Enterprise when running on Microsoft Azure by using Azure File. Operators can
+either pre-provision Azure File Shares to be consumed by Kubernetes Pods or can
+you use the Azure Kubernetes integration to dynamically provision Azure File
+Shares on demand.
+
+## Pre-Requisites
+
+This guide assumes you have already provisioned a UCP environment on to
+Microsoft Azure. The Cluster must be provisioned after meeting all of the
+pre-requisites within [Install UCP on
+Azure](/ee/ucp/admin/install/install-on-azure.md).
+
+Additionally the following guide is using the Kubernetes Command Line tool `$
+kubectl` to provision Kubernetes objects within a UCP cluster. Therefore this
+tool needs to be downloaded, along with a UCP client bundle. For more
+information on configuring CLI access to UCP see [CLI Based
+Access](/ee/ucp/user-access/cli/).
+
+## Manually Provisioning Azure File
+
+An operator can use existing Azure File Shares or manually provision new ones to
+provide persistent storage for Kubernetes Pods. Azure File Shares can be
+manually provisioned in the Azure Portal, using ARM Templates or using the Azure
+Cli. In the below example we have used the Azure Cli to manually provision an
+Azure File Shares. 
+
+### Creating an Azure Storage Account
+
+The first step when manually creating an Azure File Share, is to create an Azure
```

Comment:
I just tried this on v1.14 the other day as part of a Dockercon demo and the Azure provider now supports dynamically creating a Storage Account to host the Azure Files. So you may want to call that aspect out - the pre-req is not a hard one.

Reply:
In the Dynamic section I do call out: `Kubernetes will try and use an existing Azure Storage Account if one exists in side of the Azure Resource Group, if an Azure Storage Account does not exist Kubernetes will create one. `,That pre-req is only for a manual Azure File Shares, where you are mounting an existing one, not using the Azure provider to provision.